Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM golang:alpine AS builder
 WORKDIR /src
 
 # Copy go mod and project files
-COPY go.* .
+COPY go.* ./
```

Comment:
Interesting. Does this occur build failure? It was working with no errors on amd64

Reply:
I tried on arm64 and it fails with:

```bash
Step 1/11 : FROM golang:alpine AS builder
alpine: Pulling from library/golang
6e771e15690e: Pull complete
14aa1d43e92f: Pull complete
e18364ce0d25: Pull complete
4d476c939a05: Pull complete
4f4fb700ef54: Pull complete
Digest: sha256:2d40d4fc278dad38be0777d5e2a88a2c6dee51b0b29c97a764fc6c6a11ca893c
Status: Downloaded newer image for golang:alpine
 ---> d3f5bd625c1b
Step 2/11 : WORKDIR /src
 ---> Running in 6c0220b22067
 ---> Removed intermediate container 6c0220b22067
 ---> b7387e427a20
Step 3/11 : COPY go.* .
When using COPY with more than one source file, the destination must be a directory and end with a /
```